src/lib/fetcher.ts

```diff
@@ -151,6 +155,31 @@ export default class Fetcher implements APIFetcher {
     return tags
   }
 
+  async getOrganizationIdFromSpace(): Promise<string> {
```

Comment:
For now this is only used in tests, but in the future we'll add the ability to create `Concepts` and `ConceptSchemes` and we'll probably need something similar for that, so I added it into the fetcher. Would be happy to move it to the test package though.